spec/fixtures/factories.rb

```diff
@@ -666,4 +671,21 @@
       end
     end
   end
+
+  factory :animal_disease_case, parent: :document do
+    base_path { "/animal-disease-cases-england/example-document" }
+    document_type { "animal_disease_case" }
+    transient do
+      default_metadata do
+        {
+          "disease_type" => %w[bird-flu],
+          "zone_restriction" => "no-longer-in-force",
+          "zone_type" => %w[surveillance],
+          "virus_strain" => "h5nx",
+          "opened_date" => "2022-08-18T10:12:26+00:00",
+          "closed_date" => "2022-09-18T10:12:26+00:00",
```

Comment:
Are these field names correct?

```suggestion
          "disease_case_opened_date" => "2022-08-18T10:12:26+00:00",
          "disease_case_closed_date" => "2022-09-18T10:12:26+00:00",
```